### PR TITLE
feat: add group sort in col's values and refactor tooltip's style

### PR DESCRIPTION
**Pull request by yangzhanmei**
### 🤔 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [x] Code style optimization
- [x] Refactoring
- [x] Change the UI
- [ ] Improve performance.
- [ ] Type optimization

🐛 Bug fix

- [ ] Bug fix issue #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Other

### 📝 Description
1. add group sort in col's values
2. refactor tooltip's style after antd upgrade version
### 🖼️ Screenshot
image

|  Before  |  After  |
|----|----|
|  ❌  |  ✅  |

### CODE REVIEW

File: packages/s2-core/src/utils/sort-action.ts [hunk 1]
```diff
@@ -131,3 +135,19 @@ export const handleSortAction = (params: SortActionParams): string[] => {
...
 };
+
+export const getSortTypeIcon = (
+  sortParam: SortParam,
+  isValueCell?: boolean,
+) => {
+  if (sortParam?.sortMethod) {
+    if (isAscSort(sortParam?.sortMethod)) {
+      return 'groupAsc';
+    } else if (isDescSort(sortParam?.sortMethod)) {
```
Comment (lijinke666) [inline packages/s2-core/src/utils/sort-action.ts:146]: else 可以省略

**Comment by lijinke666** [comment]
有一个 lgtm 的报错

**Comment by lijinke666** [comment]
image

File: packages/s2-core/src/cell/header-cell.ts [hunk 2]
```diff
@@ -16,12 +20,64 @@ export abstract class HeaderCell extends BaseCell<Node> {
...
 
   protected initCell() {
     this.actionIcons = [];
   }
 
+  protected isValueCell() {
+    return this.meta.key === EXTRA_FIELD;
+  }
+
+  protected handleGroupSort(event, meta) {
```
Comment (xingwanying) [inline packages/s2-core/src/cell/header-cell.ts:47]: 这堆……是不是应该挪到交互里？统一增加一个icon的交互 cc @lijinke666

**Comment by lijinke666** [comment]
@yangzhanmei CI 挂了, 修一下 lint 问题

File: packages/s2-core/src/cell/header-cell.ts (hunk 2 above)
Comment (lijinke666) [reply packages/s2-core/src/cell/header-cell.ts:47]: 后面来统一下吧, 优先级先放低

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**